modules/jwplayerRtdProvider.js

```diff
@@ -329,15 +353,30 @@ export function addOrtbSiteContent(ortb2, contentId, contentData) {
     content.id = contentId;
   }
 
-  const currentData = content.data = content.data || [];
+  if (contentUrl) {
+    content.url = contentUrl;
```

Comment:
Should this override what might be in `content` already?

Reply:
Yes! If a publisher has requested that we enrich the bid request for a given media, then they have determined that the bid request is specific to that media, therefore we should override existing data in that field. 

Unfortunately the oRTB spec only has room for 1 content description, which is unrealistic given that multiple videos could be rendering simultaneously on the page.

Reply:
@patmmccann and @dgirardi this has been address by adding configuration params